src/apis/user.api.ts

```diff
@@ -0,0 +1,8 @@
+import { UserLoginFormTypes } from '@/pages/AdminHome/AdminHome.types';
+import { axiosClient } from './axios';
+
+export const login = async (loginData: UserLoginFormTypes) => {
+  const response = await axiosClient.post('/users/login', loginData);
+
+  return response.data;
```

Comment:
loginData는 어떤 값인지 알기가 어려운 것 같아서 userInfo와 같은 느낌도 괜찮을 것 같고 취향에 따라 구조분해할당을 해줘도 괜찮을 것 같네요!
```suggestion
export const login = async (userInfo: UserLoginFormTypes) => {
  const { data } = await axiosClient.post('/users/login', loginData);

  return data
```